src/Simulation/EntryPointDriver/Options.cs

```diff
@@ -80,5 +80,36 @@ public static Option<T> CreateOption<T>(string name, string description)
                     Arity = ArgumentArity.OneOrMore
                 }
             };
+
+        /// <summary>
+        /// Creates a prefix and alias tuple from a string representing an option alias provided through the command line
+        /// following the convention that muti-character aliases start with '--' and single-character aliases start with '-'.
+        /// </summary>
+        /// <param name="commandLineAlias">Option alias provided through the command line.</param>
+        /// <returns>TODO.</returns>
+        internal static Tuple<string, string> PrefixAliasTuple(string commandLineAlias)
```

Comment:
```suggestion
        internal static (string, string) PrefixAliasTuple(string commandLineAlias)
```

Reply:
Your suggestion of using the `RawAliases` property made this no longer needed :)